src/Http/Session.php

```diff
@@ -447,7 +448,23 @@ public function read(?string $name = null)
             return $_SESSION ?: [];
         }
 
-        return Hash::get($_SESSION, $name);
+        return Hash::get($_SESSION, $name, $default);
+    }
+
+    /**
+     * Returns given session variable, or throws Exception if not found.
+     *
+     * @param string $name The name of the session variable (or a path as sent to Hash.extract)
+     * @throws \RuntimeException
+     * @return array|string|null
```

Comment:
One could write and read integer/float too isn't?

Reply:
Yes, I copy and pasted this from the read() method. I guess both should be mixed|null then.

Reply:
Just `mixed`. `mixed` includes `null`. https://wiki.php.net/rfc/mixed_type_v2#nullability